From 57d70412cd453d5aae59a22a15febadd4f698af6 Mon Sep 17 00:00:00 2001
From: "Ho, Yu Xin" <yu.xin.ho@intel.com>
Date: Mon, 5 Sep 2016 16:30:23 +0800
Subject: [PATCH 4698/4706] ASoc: Intel: Skylake: Resolve compilation warning

commit dbf6ff4d82351650d54f3ebbb0876e1b12fbd427 from
git://git.yoctoproject.org/linux-yocto-4.1

This commit attempts to resolve the compilation warnings.

Signed-off-by: Ho, Yu Xin <yu.xin.ho@intel.com>
---
 sound/soc/intel/boards/apli_lhcrb_aic3107M.c |   23 -----------------------
 sound/soc/intel/boards/apli_lhcrb_aic3107S.c |   23 -----------------------
 2 files changed, 0 insertions(+), 46 deletions(-)
 mode change 100755 => 100644 sound/soc/intel/boards/apli_lhcrb_aic3107M.c
 mode change 100755 => 100644 sound/soc/intel/boards/apli_lhcrb_aic3107S.c
 mode change 100755 => 100644 sound/soc/intel/boards/apli_lhcrb_dummy.c
 mode change 100755 => 100644 sound/soc/intel/boards/apli_lhcrb_wm8731.c

diff --git a/sound/soc/intel/boards/apli_lhcrb_aic3107M.c b/sound/soc/intel/boards/apli_lhcrb_aic3107M.c
old mode 100755
new mode 100644
index ac80546..ac3aa58
--- a/sound/soc/intel/boards/apli_lhcrb_aic3107M.c
+++ b/sound/soc/intel/boards/apli_lhcrb_aic3107M.c
@@ -134,29 +134,6 @@ static const struct snd_soc_dapm_route apli_lhcrb_aic3107_map[] = {
 	{"ssp3 Rx", NULL, "SSP4 Mic"},
 };
 
-static int tlv320aic3107_init(struct snd_soc_pcm_runtime *rtd)
-{
-	struct snd_soc_card *card = rtd->card;
-	struct device_node *np = card->dev->of_node;
-	int ret;
-
-	/* Add specific widgets */
-	snd_soc_dapm_new_controls(&card->dapm, apli_widgets,
-				  ARRAY_SIZE(apli_widgets));
-
-	if (np) {
-		ret = snd_soc_of_parse_audio_routing(card, "ti,audio-routing");
-		if (ret)
-			return ret;
-	} else {
-		/* Set up specific audio path apli_lhcrb_aic3107_map */
-		snd_soc_dapm_add_routes(&card->dapm, apli_lhcrb_aic3107_map,
-					ARRAY_SIZE(apli_lhcrb_aic3107_map));
-	}
-
-	return 0;
-}
-
 static int apli_aic3107_hw_params(struct snd_pcm_substream *substream,
 	struct snd_pcm_hw_params *params)
 {
diff --git a/sound/soc/intel/boards/apli_lhcrb_aic3107S.c b/sound/soc/intel/boards/apli_lhcrb_aic3107S.c
old mode 100755
new mode 100644
index 952eb01..a8c6362
--- a/sound/soc/intel/boards/apli_lhcrb_aic3107S.c
+++ b/sound/soc/intel/boards/apli_lhcrb_aic3107S.c
@@ -134,29 +134,6 @@ static const struct snd_soc_dapm_route apli_lhcrb_aic3107_map[] = {
 	{"ssp3 Rx", NULL, "SSP4 Mic"},
 };
 
-static int tlv320aic3107_init(struct snd_soc_pcm_runtime *rtd)
-{
-	struct snd_soc_card *card = rtd->card;
-	struct device_node *np = card->dev->of_node;
-	int ret;
-
-	/* Add specific widgets */
-	snd_soc_dapm_new_controls(&card->dapm, apli_widgets,
-				  ARRAY_SIZE(apli_widgets));
-
-	if (np) {
-		ret = snd_soc_of_parse_audio_routing(card, "ti,audio-routing");
-		if (ret)
-			return ret;
-	} else {
-		/* Set up specific audio path apli_lhcrb_aic3107_map */
-		snd_soc_dapm_add_routes(&card->dapm, apli_lhcrb_aic3107_map,
-					ARRAY_SIZE(apli_lhcrb_aic3107_map));
-	}
-
-	return 0;
-}
-
 static int apli_aic3107_hw_params(struct snd_pcm_substream *substream,
 	struct snd_pcm_hw_params *params)
 {
diff --git a/sound/soc/intel/boards/apli_lhcrb_dummy.c b/sound/soc/intel/boards/apli_lhcrb_dummy.c
old mode 100755
new mode 100644
diff --git a/sound/soc/intel/boards/apli_lhcrb_wm8731.c b/sound/soc/intel/boards/apli_lhcrb_wm8731.c
old mode 100755
new mode 100644
-- 
1.7.5.4

